app/helpers/enterprises_helper.rb

```diff
@@ -21,6 +21,7 @@ def available_shipping_methods
   def available_payment_methods
     return [] unless current_distributor.present?
     payment_methods = current_distributor.payment_methods.available(:front_end).all
+    payment_methods.reject!{ |p| p.type.ends_with? "StripeConnect" } unless Spree::Config.stripe_connect_enabled
```

Comment:
This is the second time I see this, so before duplicating this everytime we show payment methods, I suggest adding a scope/class method on our decorator that filters it out depending on `stripe_connect_enabled`.

Reply:
This logic only appears in one place now, and I don't think it is worth adding a dedicated scope. I will however extract all of the logic for finding 'available' payment methods to a service object.